server/api/cvrs.py

```diff
@@ -857,21 +857,25 @@ def parse_hart_cvrs(
     """
     A Hart CVR export is a ZIP file containing an individual XML file for each ballot's CVR.
 
-    Either a single ZIP file can be provided or multiple, one for each tabulator. The latter is
-    necessary when batch names are not unique across tabulators. Hart XML files don't contain
-    tabulator info, but Hart does allow exporting CVRs by tabulator. When multiple files are
-    provided, we use zip file names (with ".zip" removed) as tabulator names. (These tabulator
-    names need to match the ballot manifest, of course.)
+    Either a single ZIP file can be provided or multiple, one for each tabulator. When multiple
+    files are provided, the ZIP file names (with ".zip" removed) will be used as tabulator names.
 
-    Separate from the ZIP file(s), an optional scanned ballot information CSV can be provided. If
-    provided, the `UniqueIdentifier`s in it will be used as imprinted IDs. Otherwise, `CvrGuid`s
-    will be used.
+    Separate from the ZIP files, optional scanned ballot information CSVs can be provided. If
+    provided, the "Workstation" values in them will be used as tabulator names, and the
+    "UniqueIdentifier" values in them will be used as imprinted IDs. Otherwise, "CvrGuid" values
+    will be used as imprinted IDs.
+
+    If both multiple ZIP files are provided and scanned ballot information CSVs are provided, the
+    ZIP file names will take precedence over the "Workstation" values as tabulator names.
```

Comment:
I had to pick one method to take precedence and landed on higher precedence for the multiple ZIP files, since ZIP file names are easier to update than scanned ballot information CSVs